doc/verilog_systemverilog.txt

```diff
@@ -448,6 +448,19 @@ Example:
     Note: The commands |:VerilogFoldingAdd| and |:VerilogFoldingRemove| are
     provided to allow an easier management of this variable.
 
+                                                      *g:verilog_quick_syntax*
+g:verilog_quick_syntax~
+Default: undefined
+
+    Whena enabled, syntax regions will not be defined with Vim's 'syntax'
+    command. This can help performance if you are using an old machine or
+    veiwing a large file such as a netlist. This can be useful if you don't
+    care about automatic indentation but still wish to have syntax coloring.
```

Comment:
(Still did not have the opportunity to look into your code, so I don't know if the following comment makes sense.)

Would it be possible to have a syntax based indentation script that is enabled with a global variable? This way we could keep your old indentation method using a "quick syntax and automatically disable the "quick syntax" when the new indentation method is used. This would obviously result in replacing `g:verilog_quick_syntax` with something like `g:verilog_syntax_based_indentation`.